partner_risk_insurance/README.rst

```diff
@@ -0,0 +1,85 @@
+======================
+Partner Risk Insurance
+======================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/github-OCA%2Fcredit--control-lightgray.png?logo=github
+    :target: https://github.com/OCA/credit-control/tree/11.0/partner_risk_insurance
+    :alt: OCA/credit-control
+.. |badge3| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/credit-control-11-0/credit-control-11-0-partner_risk_insurance
+    :alt: Translate me on Weblate
+.. |badge4| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/262/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| 
+
+This module adds a new tab in the partner form to introduce risk
+insurance information.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module you need to:
+
+#. Go to *Sales > Customer > Create*.
+#. Go to *Credit Insurance* tab.
+#. Set insurance risk options.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/credit-control/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/credit-control/issues/new?body=module:%20partner_risk_insurance%0Aversion:%2011.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* OdooMRP team
+* AvanzOSC
+* Tecnativa - Pedro M. Baeza
```

Comment:
Is this line to be recreated with this new value? https://github.com/OCA/credit-control/pull/7/files#diff-0f1fbcd1c48ec7fa07c7af50d7d6d1c1R8

Reply:
Yes, it will be changed after merge.